### PR TITLE
Update README.md

### LINKED ISSUE #115: gss README is out of date

### PR DESCRIPTION
**Pull request by singular-value**
Closes #115

### CODE REVIEW

**Review by paaige — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # general-superstaq
-This package is used to house endpoints for the finance and logistics applications for SuperstaQ.
\ No newline at end of file
+This package is used to house code shared between the superstaq clients ([cirq-superstaq](https://github.com/SupertechLabs/cirq-superstaq) and [qiskit-superstaq](https://github.com/SupertechLabs/qiskit-superstaq)).
```
Comment (vtomole) [inline README.md:2]: SupermarQ import gss as well. Maybe we generalize this sentence to cover all our repos and future repos as well.

Comment (singular-value) [reply README.md:2]: a link to ss would break for external users though.

Comment (vtomole) [reply README.md:2]: How about: "This package is used to house code shared between all of Super.tech's codebase like the clients ([cirq-superstaq](https://github.com/SupertechLabs/cirq-superstaq) and [qiskit-superstaq](https://github.com/SupertechLabs/qiskit-superstaq)), [SupermarQ](https://github.com/SupertechLabs/SupermarQ), and more."

Comment (singular-value) [reply README.md:2]: ah right SupermarQ

Comment (singular-value) [reply README.md:2]: Well...actually SupermarQ doesn't (directly) use gss.

Comment (vtomole) [reply README.md:2]: It uses the checks. Maybe that doesn't count?

Comment (singular-value) [reply README.md]: @vtomole  updated

**Pull request merged**